runtime/src/bank.rs

```diff
@@ -2039,7 +2044,16 @@ impl Bank {
     /// vote accounts for the specific epoch along with the stake
     ///   attributed to each account
     pub fn epoch_vote_accounts(&self, epoch: Epoch) -> Option<&HashMap<Pubkey, (u64, Account)>> {
-        self.epoch_stakes.get(&epoch).map(Stakes::vote_accounts)
+        self.epoch_stakes
+            .get(&epoch)
+            .map(|x| Stakes::vote_accounts(&x.0))
+    }
+
+    pub fn total_epoch_stake(&self) -> u64 {
+        self.epoch_stakes
```

Comment:
Are these keyed by vote identity, or node identity?

Reply:
These are keyed by vote identity because thats what the gossip listener is getting from votes on gossip